apps/comments-ui/src/App.tsx

```diff
@@ -224,7 +224,7 @@ const App: React.FC<AppProps> = ({scriptTag}) => {
             <CommentsFrame ref={iframeRef}>
                 <ContentBox done={done} />
             </CommentsFrame>
-            {state.comments.length > 0 ? <AuthFrame adminUrl={options.adminUrl} onLoad={initAdminAuth}/> : null}
+            {state.initStatus === 'success' && <AuthFrame adminUrl={options.adminUrl} onLoad={initAdminAuth}/>}
```

Comment:
I first tried always rendering `<AuthFrame` but that lead to a stuck loading state. Perhaps a more involved change is required to avoid this conditional render entirely.